EntityModel/Migrations/20190228215829_InitialCreate.Designer.cs

```diff
@@ -1,94 +0,0 @@
-﻿// <auto-generated />
```

Comment:
You'll want to not delete these files. This is the migration from before the hackathon started. You can just revert the changes to these 3

Reply:
Sorry @scfi0 . It was supposed to be `git checkout origin/develop -- path/to/file`. My bad 😅 :